start-server.sh

```diff
@@ -15,14 +15,25 @@
 # limitations under the License.
 
 #provide the path to Node directory before starting the run
-#NODE_PATH=/opt/local/node-v6.10.0-linux-x64/
-if [ "x$NODE_PATH" == "x" ]; then
-	echo "Node path is not set"
+export PATH=/opt/local/node-v6.10.0-linux-x64/bin:$PATH
+
+IMAGE_NAME=""  #if not set, workload uses image.jpeg by default
+
+if [ ! type npm ] || [ ! type node ]; then
```

Comment:
You need to remove the []'s. The `[` program (see `which [`) or builtin runs what you placed inside and set the exit code ($?) accordingly (0 to true, 1 for false). `if` checks that exit code. Since we only care about the exit code, not the output, we don't need `[`.